htdocs/survey.php

```diff
@@ -94,14 +94,15 @@ function initialize()
         if (empty($this->TestName) && empty($this->CommentID)) {
             throw new Exception("Data has already been submitted.", 403);
         }
-        $pageNum = null;
 
+        $pageNum = null;
         if (!empty($_REQUEST['pageNum'])) {
             $pageNum = $_REQUEST['pageNum'];
         }
 
         if ($pageNum === 'finalpage') {
             $this->Subtest = 'finalpage';
+            $pageNum       = -1;
```

Comment:
I don't understand why you're making this change. It looks like it'll break the page numbering on the review page.

Reply:
You're right, the change I did in [getNextPageNum()](https://github.com/aces/Loris/blob/a96e089e3d5f9f9b93341b98ea6cbb290acf8b55/htdocs/survey.php#L152) and [here](https://github.com/aces/Loris/blob/a96e089e3d5f9f9b93341b98ea6cbb290acf8b55/htdocs/survey.php#L129) makes that change not necessary anymore.